apps/user/models/user.go

```diff
@@ -26,6 +39,20 @@ type User struct {
 	Profiles           []UserProfile
 }
 
+//BeforeSave ...
+func (user *User) BeforeSave() {
+	var timeconfig []TimeProviderConfig
+
+	user.FirstName = strings.TrimSpace(user.FirstName)
```

Comment:
Please, make a model method for this and Also, look for other places.

Reply:
done

Reply:
That's not a model method please check once more.

Reply:
done